config/environments/production.rb

```diff
@@ -100,4 +100,10 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.middleware.use ExceptionNotification::Rack,
+    email: {
+      sender_address: %{'st-pax' <transit-it@admin.umass.edu>},
```

Comment:
Is this missing a `w` for the `%w`? Also, should we be using `%w{}` instead of `%w[]`?

Reply:
Nope, it's that way in round-three and the exception_notifier docs. It resolves to a string, not an array of strings.

Reply:
https://github.com/smartinez87/exception_notification